app/views/events/_form.html.erb

```diff
@@ -1,38 +1,40 @@
-<%= form_with model: @event, local: true do |form| %>
-  <% if @event.errors.any? %>
-    <div id="error_explanation">
-      <h2>
-        <%= pluralize(@event.errors.count, "error") %> prohibited
-        this article from being saved:
-      </h2>
-      <ul>
-        <% @event.errors.full_messages.each do |msg| %>
-          <li><%= msg %></li>
-        <% end %>
-      </ul>
+<div class="container col-3">
```

Comment:
Let's discuss when we have meeting next time and I will guide you about the bootstrap classes and where and when to use which classes and which classes we avoid using together.
as here Container is to contain other things.

and `col` classes are meant to be used under `row` class.